llvm/tools/llvm-exegesis/lib/Target.h

```diff
@@ -86,6 +86,16 @@ class ExegesisTarget {
                 ArrayRef<const char *> ValidationCounters,
                 const pid_t ProcessID = 0) const;
 
+  virtual bool checkOpcodeSupported(int Opcode,
+                                    const MCSubtargetInfo &SI) const {
+    return true;
```

Comment:
Should this default to false? If we don't know what target or opcode we are dealing with, I would assume that false is the default to go for. Similarly to returning NoRegister by default below

Reply:
Addressed